### DISCUSSION TITLE
Getting Fetch error while adding Datasource

### DISCUSSION THREAD

**Discussion by navimon** [grafana/grafana-infinity-datasource#471] [Q&A]
Hi I tried installing your plugin using the zip file by running the command:
`grafana-cli --pluginUrl /tmp/yesoreyeram-infinity-datasource-1.2.0.zip plugins install yesoreyeram-infinity-datasource`

However when I try to create a datasource of type infinity i get the following error:
`Fetch error: 404 Not Found Instantiating http://localhost:8080/public/@emot
<img width="1471" alt="Screenshot 2022-11-21 at 7 24 48 PM" src="https://user-images.githubusercontent.com/19780750/203072540-76cc0544-d201-4c78-abc9-4f3a2dccfff0.png">
ion/css Loading http://localhost:8080/public/plugins/yesoreyeram-infinity-datasource/module.js Loading plugins/yesoreyeram-infinity-datasource/module`

GRAFANA version is 7.5.10
Infinity version is 1.2.0 / 1.1.0

**Comment by navimon** [comment]
<img width="1216" alt="Screenshot 2022-11-24 at 2 40 46 PM" src="https://user-images.githubusercontent.com/19780750/203743305-12378d7d-54ef-4098-9774-5f24ddeb5469.png">

**Comment by navimon** [comment]
![Screenshot 2022-11-24 at 2 43 13 PM](https://user-images.githubusercontent.com/19780750/203743876-bae3da10-9d07-48cc-b5c7-4c319458d262.png)

**Comment by yesoreyeram** [comment]
Use full internet url for the plugin url. Refer docs on grafana about installing plugin different way.
